components/landing/TheQuickStart/PrerequisitesForMac.vue

```diff
@@ -1,33 +1,16 @@
 <template>
   <div class="prerequisites">
     <div class="prerequisites__section">
+      <p>Run the following command and follow the onscreen instructions:</p>
+      <pre class="code prerequisites__code">curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh</pre>
+
       <p>
-        To use the Clang compiler on macOS, you need to install an extra library
-        for supporting OpenMP. You can use
-        <AppLink url="https://brew.sh/">
-          brew
-        </AppLink>
-        to install this and other dependencies.
+        See <AppLink url="https://rustup.rs/">
```

Comment:
It would be nice to have the links tracked with segment

Reply:
Added segment info in this commit  https://github.com/Qiskit/qiskit.org/pull/2904/commits/e4f624245eefd7dc5dc2fbe21522256669d6e091